consai_examples/consai_examples/field_observer.py

```diff
@@ -764,3 +764,56 @@ def _forward_robots_id(self, my_robot_id, my_robot_pos, robots_pos, robots_vel,
 
         return forward_robots_id
 
+    # シューターが相手ゴールに向かってシュートできるかを判定する関数
+    def get_shoot_point(self, my_robot_id):
+        # ゴールポストのx座標
+        goal_post_x = 6
+        # ゴールポストのy座標
+        goal_post_y = 0.9
+        # フィールド中央から見たゴールポストの座標
+        goal_post = [[goal_post_x, goal_post_y], [goal_post_x, - goal_post_y]]
+        # シュートを打つ目標点の座標
+        goal_center_target = [goal_post_x, 0]
+        # ロボットの実直径
+        robot_diameter = 0.2
+        # ロボットの想定移動時間（移動しないものとして扱う）
+        dt = 0
+
+        # 味方ロボットの位置、相手ロボットの位置と速度を取得
+        our_robots_pos = self.our_robots_pos
```

Comment:
self.our_robots_posは、この関数の裏で更新されるので、処理中に値が変わる可能性があります。
（pythonの`=`は代入ではなく参照渡しなので）

copy.deepcopyを使用して、コピーしましょう。

```suggestion
        import copy
        OUR_ROBOTS_POS = copy.deepcopy(self.our_robots_pos)
```